gax/src/main/java/com/google/api/gax/retrying/BasicRetryingFuture.java

```diff
@@ -166,6 +170,18 @@ void handleAttempt(Throwable throwable, ResponseT response) {
             retryAlgorithm.createNextAttempt(throwable, response, attemptSettings);
         boolean shouldRetry = retryAlgorithm.shouldRetry(throwable, response, nextAttemptSettings);
         if (shouldRetry) {
+          // Log retry info
+          if (LOG.isLoggable(Level.FINEST)) {
+            LOG.log(
```

Comment:
The same logic is used for polling (LRO operations). Please try to formulate logging in a form, which fits both (i.e. instead of "retryDelay" just "delay" and stuff like that).

Reply:
done